src/frequenz/channels/_base_classes.py

```diff
@@ -169,11 +169,13 @@ def __init__(self, recv: Receiver[T], transform: Callable[[T], U]) -> None:
 
         Args:
             recv: The input receiver.
-            transform: The function to run on the input
-                data.
+            transform: The function to run on the input data.
         """
         self._recv = recv
+        """The input receiver."""
```

Comment:
```suggestion
        """The input channel receiver."""
```

Reply:
I think just receiver is fine, as it could not be from a channel (like `Timer` or `FileWatcher`).